### ISSUE TITLE
some issues about pc2cad

### ISSUE THREAD

**Issue by LJXCMQ** [ChrisWu1997/DeepCAD#23]
Hi,@ChrisWu1997 ：
Your work is excellent and I am very interested in your paper on point cloud reconstruction engineering with Pointnet++. So there are some training details I would like to ask for:
![image](https://github.com/ChrisWu1997/DeepCAD/assets/110969883/e96210b5-9199-405c-b5ef-3774ee680ca8)
I got the same loss curve as you uploaded in the previous comment, although I found that the loss drops below 0.07, but it didn't work out so well. In particular, the difference between the decoded CAD model and the Ground Truth is so great that it can be observed with eyes.
So I'd like to ask for more details about the training:
I'm directly using the Z obtained from the provided encoder in the pre-trained model;
The point cloud file comes from a series of conversions, which are: json->step->stl->ply, because otherwise, using the cad2pc program directly would consist of close to half of the data that cannot be converted.
I didn't normalize any of this data, does it matter?

**Comment by ybdog** [comment]
Could I ask what the purpose of pc2cad is and how to use it?
